x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/event_filters_translations.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+
+export const POLICY_ARTIFACT_EVENT_FILTERS_LABELS = Object.freeze({
+  deleteModalTitle: i18n.translate(
+    'xpack.securitySolution.endpoint.policy.eventFilters.list.removeDialog.title',
+    {
+      defaultMessage: 'Remove event filter from policy',
+    }
+  ),
+  deleteModalImpactInfo: i18n.translate(
+    'xpack.securitySolution.endpoint.policy.eventFilters.list.removeDialog.messageCallout',
+    {
+      defaultMessage:
+        'This event filter will be removed only from this policy and can still be found and managed from the artifact page.',
+    }
+  ),
+  deleteModalErrorMessage: i18n.translate(
+    'xpack.securitySolution.endpoint.policy.eventFilters.list.removeDialog.errorToastTitle',
+    {
+      defaultMessage: 'Error while attempt to remove event filter',
+    }
+  ),
+  flyoutWarningCalloutMessage: i18n.translate(
```

Comment:
Wondering if this one should be a function that accepts on input the max number of items to display - since you created a const. then use that in the message